### DISCUSSION TITLE
Attempting to timeout a user, returns "Missing Permission" error (Permission confirmed)

### DISCUSSION THREAD

**Discussion by dynamojake** [Rapptz/discord.py#9417] [Q&A] [answered]
I apologise. As it turns out, I was correct and it was nothing more than a case of me being an idiot. While I had double checked permission structure & roles, I had overlooked the fact that the test user had been made server owner.

**Comment by tailoric** [comment, marked as answer]
Biggest reason i could think off you are either trying to timeout the owner of a server or someone who has a higher role than your bots highest role

**Reply by dynamojake** [reply]
Sadly, it wouldn't appear to be that simple.
Currently, I'm testing with the Bot using Admin permissions and the user with none. The Bot's role is also at the top of the hierarchy. 
Thanks for the speedy reply though <3
